File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int, int> mp;
        for(int num: nums){
            mp[num]++;
        }

        multimap<int, int, std::greater<int>> freq;
        for(auto& elem: mp){
            freq.insert({elem.second, elem.first});
        }

        vector<int> returnVec;
        auto it = freq.begin();
        for(int i = k; i>0; i--){
            returnVec.push_back(it->second);
            it++;
        }

        return returnVec;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        num_set = set(nums)

        currLong = 0
        for num in num_set:
            if((num - 1) not in num_set):
                count = 1
                while(num + count in num_set):
                    count += 1
                if(count > currLong):
                    currLong = count
        
        return currLong

            
        